docs/application/tizen-studio/release-notes/4-1-release-notes.md

```diff
@@ -0,0 +1,59 @@
+# Tizen Studio 4.1 Release Notes
+
+- Release Date: Dec 21, 2020
+
+## IDE and Tools
+
+### New Features
+
+- IDE
+  - Tizen Mobile/Wearable 5.5 emulator image has been updated.
+  - IoT Setup Manager has been updated.
+
+
+### Deprecated
+   - Tizen RT has been deprecated from Tizen Studio 4.1.
```

Comment:
Should it read, 
  - Tizen RT IDE has been deprecated from Tizen Studio 4.1.
  
or

  - Tizen Studio for RT has been deprecated from Tizen Studio 4.1.

OR
 
-  Support for Tizen RT has been deprecated from Tizen Studio 4.1.

Reply:
done